shared/src/main/scala/amf/core/rdf/parsers/CustomPropertiesParser.scala

```diff
@@ -43,7 +43,7 @@ class CustomPropertiesParser(linkFinder: NodeFinder, sourcesRetriever: SourcesRe
 
                 new DynamicTypeParser(linkFinder, sourcesRetriever).parse(entries.head).foreach { pn =>
                   extension.withId(pn.id)
-                  extension.withExtension(pn)
+                  extension.set(DomainExtensionModel.Extension,pn, Annotations.inferred())
```

Comment:
same comment as above, can't the "withExtension" accept an Annotations object with default empty annotations?

Reply:
Could, but keep in mind that is no an empty annotation.
Annotations.inferred() is equivalent to Annotations(Inferred())
On the other hand, I think all the "with" method the build inners object should disappears, as the purpose of those were to add the new object to the parent fields and adopt it.
Once the final adoption is done, this will no longer necessary.